### ISSUE TITLE
Do not warn for "macro in comment" for special comments like #!BuildIgnore

### ISSUE THREAD

**Issue by darix** [rpm-software-management/rpmlint#1069]
```
#!BuildIgnore:  %pgname
```

is a valid construct in this case.

**Comment by danigm** [comment]
PR created to do not check for macros in comments that starts with `#!`. I've also added a test to verify this.
